projects/04_cube_xr/main.cpp

```diff
@@ -317,9 +317,8 @@ void ProjApp::Render()
         float4x4 P = glm::perspective(glm::radians(60.0f), GetWindowAspect(), 0.001f, 10000.0f);
         float4x4 V = glm::lookAt(float3(0, 0, 0), float3(0, 0, 1), float3(0, 1, 0));
 
-        // No need to wait for imageAcquiredFence since xrWaitSwapchainImage is called in AcquireNextImage.
-        if (GetSettings()->enableXR) {
-            P = GetXrComponent().GetProjectionMatrixForCurrentView(0.001f, 10000.0f);
+        if (IsXrEnabled()) {
+            P = GetXrComponent().GetProjectionMatrixForCurrentView();
```

Comment:
as mentioned in the other comment, i think it is still better to pass the near and far as param to be consistent with the non-xr code (see line 317 in this file)

Reply:
See comment above about this issue.